### ISSUE TITLE
hey plss help me

### ISSUE THREAD

**Issue by subha3022** [konhito/AI_IMAGE-GENERATION#4]
plss help me i cant slove the bug  there is something alert like that  " Incorrect is not a valid JSON " plss help i beg you

**Comment by konhito** [comment]
on your terminal mongo db is connected?

**Comment by subha3022** [comment]
yes, plss can you solve this in meet i cant understand plss i have to make this project immediately

**Comment by subha3022** [comment]
hey are you there

**Comment by subha3022** [comment]
hey plss help

**Comment by konhito** [comment]
i would love to but this project is broken even if i fix that problem still i cant make it work properly
follow sonny sanga ai image generation project that works

**Comment by konhito** [comment]
im also new u need someone experience to fix those issues

**Comment by subha3022** [comment]
bhaiya i invest 3 days on it plss help fix the only issue other wise i will manage plss

**Comment by konhito** [comment]
im in college rn send me your code ill review it

**Comment by subha3022** [comment]
honestly i dont know much about github how to send via  github i would really helpful if after you come back you send me meetlink i share my screen and you help plsss

**Comment by konhito** [comment]
cool

**Comment by subha3022** [comment]
i promise after that i never disturb you

**Comment by konhito** [comment]
nah its ok im also new if i have so doubt u can clear it.....thats why its open source we all can learn together by connecting ppls

**Comment by subha3022** [comment]
thanks bhaiya ... sure i always there for u .. when will you come back !

**Comment by konhito** [comment]
ill send u meet link around 4-5

**Comment by subha3022** [comment]
ok bhaiya here ??

**Comment by konhito** [comment]
hey u there

**Comment by konhito** [comment]
send me meet link rn

**Comment by subha3022** [comment]
ok bhaiya yes 1 min

**Comment by subha3022** [comment]
[https://meet.google.com/uwb-chfx-wkr](url)

**Comment by subha3022** [comment]
this is the link bhaiya

**Comment by konhito** [comment]
cool im coming

**Comment by harshit-333** [comment]
Hey, do you know where's the problem when the posts are not shown on the showcase community even it's showing on cloudinary. A little help would be appreciated!

**Comment by konhito** [comment]
Sure can you share your code so,I  can look into it

**Comment by harshit-333** [comment]
Thanks for your reply, but it's solved now. The problem was somewhere in my
mongodb server!
Again thanks for your concern.
